src/main/kotlin/Calculator.kt

```diff
@@ -0,0 +1,73 @@
+class Calculator() {
+    fun calculate(expression: String?): Int {
+        nullOrEmptyCheck(expression)
+
+        val arr_experssion: Array<String> =
+            expression?.split(" ".toRegex())?.dropLastWhile { it.isEmpty() }?.toTypedArray() as Array<String>
+        val operando1 = arr_experssion[0].toInt()
+
+        val result = operatorCheck(operando1, arr_experssion)
+
+        return result
+    }
+
+    fun operatorCheck(operando1: Int, arr_fomula: Array<String>): Int {
```

Comment:
코드 전체적으로 접근 제어자를 신경 써주시면 더욱 좋을 거 같아요~
```suggestion
    private fun operatorCheck(operando1: Int, arr_fomula: Array<String>): Int {
```